src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -138,6 +138,50 @@ export default class DefaultAudioVideoController
     this._audioMixController = new DefaultAudioMixController(this._logger);
     this.meetingSessionContext.logger = this._logger;
     this._eventController = new DefaultEventController(this);
+
+    // To log packets per second numbers, set
+    // ```
+    // window.CHIME_LOG_PPS = true;
+    // ```
+    // before creating the AV controller.
+    //
+    // This is not an ongoing supported interface, but it is useful for testing.
+
+    /* istanbul ignore next */
+    if (((global as unknown) as { CHIME_LOG_PPS: boolean }).CHIME_LOG_PPS) {
```

Comment:
Could we avoid this in `DefaultAudioVideoController` and add in the demo itself? We have done earlier at few places like: [1](https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.ts#L1083), [2](https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.ts#L1053).

Reply:
I'm not sure if I was able to get it to work without poking directly at the sender. I want customers to have an easy way to determine whether they are affected by this issue. If we can make it a handful of lines of code to add, then sure, but otherwise I think it makes sense to be in core.